konfera/templates/konfera/event_speakers.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'konfera/base.html' %}
+
+{% block title %}<h1>{{ event }} Speakers</h1>{% endblock %}
```

Comment:
`<title>{% block title %}BASE TEMPLATE{% endblock %}</title>` works for me.

konfera/templates/konfera/event_sponsors.html
`{% extends 'konfera/base.html' %}`
`{% block title %}{{ event }}{% endblock %}`